long_term_server/AgentDescription description
---
bool success
string assigned_name
